documentation/proc-pages/eng-models/central-solenoid.md

```diff
@@ -174,3 +174,6 @@ no. 106 (`ftmargoh`).
 
 It is recommended that EITHER the temperature margin constraint (60), OR the current density 
 constraints (26 and 27) are activated.
+
+!!! note
```

Comment:
It might be better to change this to a tip and add a title.
Such as `!!! tip "Recommended maximum critical current"`

Reply:
Made change